etc/win-ci/crystal.iss

```diff
@@ -0,0 +1,335 @@
+#define MyAppName "Crystal"
+#define VersionFile FileOpen("portable\src\VERSION")
+#define MyAppVersion FileRead(VersionFile)
+#define MyAppVersionNum StringChange(MyAppVersion, "-dev", "")
+#expr FileClose(VersionFile)
+#define MyAppPublisher "Manas Technology Solutions"
+#define MyAppURL "https://crystal-lang.org/"
+#define MyAppExeName "crystal.exe"
+#define MyAppAssocName MyAppName + " Source File"
+#define MyAppAssocExt ".cr"
+#define MyAppAssocKey StringChange(MyAppAssocName, " ", "") + MyAppAssocExt
+
+[Setup]
+AppId={{7C307DDF-447E-46C5-BB3B-47A6F652D7C8}
+AppName={#MyAppName} x86_64-windows-msvc
+AppVersion={#MyAppVersion}
+AppCopyright=Copyright 2012-2023 Manas Technology Solutions

```

Comment:
How do we ensure the year is kept up to date?

Reply:
If I could not find a way in Inno Setup to understand `SOURCE_DATE_EPOCH`, then manually. I know this is not ideal but `NOTICE.md` also needs to be updated manually for the same reason

Reply:
Actually never mind, we could simply extract this from `crystal.exe`'s manifest